tests/includes/awscli.sh

```diff
@@ -0,0 +1,21 @@
+setup_awscli_credential() {
+	if ! which aws >/dev/null 2>&1; then
+		sudo snap install aws-cli --classic || true
```

Comment:
I don't like this, but since its a snap, its easy to remove.

Reply:
We install other snaps in other bash ci tests, so this is following accepted practice.

Reply:
The snap installing is fine, its more that we haven't yet added protection from messing up someone's machine. We can deal with this another time, but ideally build scripts should know when they are allowed to mutate the machine they are run on.